README.md

```diff
@@ -41,6 +41,89 @@ evolution of the schema. The schema follows [semver](https://semver.org/) versio
 
 This version will not always match the npm package version, as some changes to the npm package might not have changes to the schema. We will publish a list of schema versions and their associated npm versions and git tags.
 
+## Example
+
+Given the following source code:
+
+```js
+class MyElement extends HTMLElement {
+  static get observedAttributes() {
+    return ['disabled'];
+  }
+
+  set disabled(val) {
+    this.__disabled = val;
+  }
+  get disabled() {
+    return this.__disabled;
+  }
+
+  fire() {
+    this.dispatchEvent(new Event('disabled-changed'));
+  }
+}
+
+customElements.define('my-element', MyElement);
+```
+
+The output manifest would look like:
+```json
+{
+  "schemaVersion": "1.0.0",
+  "readme": "",
+  "modules": [
+    {
+      "kind": "javascript-module",
+      "path": "my-project/my-element.js",
+      "declarations": [
+        {
+          "kind": "class",
+          "description": "",
```

Comment:
It'd be nice to add one jsdoc comment somewhere, on the class or a method, and show it in the output.